### ISSUE TITLE
0022-ExceptionHandling.st.html is broken

### ISSUE THREAD

**Issue by codefrau** [pablomarx/Smalltalk76#1]
It's missing the last method, and the HTML trailer

**Comment by pablomarx** [comment]
The restore_alto_tools command complains about this file.  I just didn't notice at the time:

```
$ restore_alto_files --bravo=ExceptionHandling.st 
restore_alto_files of 20 June 2017
  Command-line options: --bravo=ExceptionHandling.st 
*** Missing CR in Bravo format file: ExceptionHandling.st
$ ls -l ExceptionHandling.st
-rw-r--r--  1 swhite  staff  11063 Feb  6 12:03 ExceptionHandling.st
```

I'll try to investigate this further in the next couple of days.

**Comment by pablomarx** [comment]
@bertfreudenberg I believe this is fixed now.  Much appreciate you bringing it to my attention!

**Comment by pablomarx** [comment]
It looks like the following files were also ending prematurely:

 * 0004-Sets-and-Dictionaries.st.html
 * 0005-Graphical-Objects.st.html
 * 0007-Windows.st.html
 * 0011-IFS-File-System.st.html
 * 0012-Events.st.html
 * 0014-Press-File-Support.st.html
 * 0019-Form-Path-Image.st.html

I merged corrections for these in PR #3

**Comment by codefrau** [comment]
Looking good!
